index.js

```diff
@@ -122,18 +122,30 @@ exports.getCommitLanguages = (visibility, token) => {
 
                 // For each file in the commit files
                 _.each(commits, (commit) => {
+                    var commitLangs = []; // Store all the languages present in commit
```

Comment:
Old habits die hard.. `var` hasn't raised any scoping issues or anything yet, but it would probably be preferred to change them all to `let`